internal-messaging/unixdomain/header/UnixDomainStreamSocketServer.h

```diff
@@ -22,7 +22,7 @@ using std::string;
 	A simple server in the internet domain using Unix Domain Socket
 */
 //REF: http://www.linuxhowtos.org/C_C++/socket.htm
-class UnixDomainStreamSocketServer : protected UnixDomainStreamSocket {
+class UnixDomainStreamSocketServer : public UnixDomainStreamSocket {
```

Comment:
Why are we removing UnixDomainStreamSocketServer permissions to access protected members of UnixDomainStreamSocket?

Reply:
returned it to protected